frontend/src/pages/ActivityReport/Pages/components/Objectives.js

```diff
@@ -34,7 +34,7 @@ export default function Objectives({
     defaultValues,
   });
 
-  const objectiveIds = fields ? fields.map(({ value }) => value) : [];
+  const objectiveIds = fields ? fields.map(({ value, id }) => id || value) : [];
```

Comment:
Is it possible to standardize the sequelize so that we don't have to "always be checking ©️" this?

Reply:
Yes I am adding value to the objective save. This also fixes the issue of Objectives not being removed from the list.